plugins/calendar/src/main/java/de/unistuttgart/iaas/amyassist/amy/plugin/calendar/CalendarSpeech.java

```diff
@@ -29,7 +29,7 @@
 
 /**
  * This is the speech class, which contains the commands for the calendar
- * 
+ *
```

Comment:
in method `getEventsToday`: Just check the last param, instead of finding out the position

Reply:
fixed